### DISCUSSION TITLE
Memory leaks using visual studio 2022

### DISCUSSION THREAD

**Discussion by v71** [DanielChappuis/reactphysics3d#337] [Q&A]
I have noticed that in debug mode, the library leaks quite a bit, this could be caused by the same visual studio, which signals a memory leak for memory allocated on the stack rahter than on the heap. Have you experienced something like this ?

**Comment by DanielChappuis** [comment]
Hello. Sorry for my very late answer but do you have more information about this memory weak ? Do you know where or which object/pointer seems to be leaking ? How did you find about this leak ? Did you have warning somewhere in Visual Studio ?
